LoveAndHiphop/LoveAndHiphop/DetailViewController.swift

```diff
@@ -32,50 +32,59 @@ class DetailViewController: UITableViewController {
     
     var userObject: UserObject?
     var likedByUsers: [String]?
+    var isMatch: Bool?
```

Comment:
When the current user and the selected users both like each other, this flag needs to be set to True. And if this flag is true, we will be showing that "its a match" icons on the Detailed view.